sql/core/src/test/resources/tpcds-plan-stability/approved-plans-v1_4/q11.sf100/explain.txt

```diff
@@ -89,394 +89,394 @@ TakeOrderedAndProject (87)
 
 
 (1) Scan parquet default.store_sales
-Output [4]: [ss_sold_date_sk#1, ss_customer_sk#2, ss_ext_discount_amt#3, ss_ext_list_price#4]
+Output [4]: [ss_sold_date_sk#x, ss_customer_sk#x, ss_ext_discount_amt#x, ss_ext_list_price#x]
```

Comment:
Sorry for bringing massive changes again.

Do you think we should avoid this change? If it should, I think we could make a custom copy of `replaceNotIncludedMsg` in `PlanStabilitySuite` instead of resue.

Reply:
I think it's fine

Reply:
After a second thought, this might be useful to understand the query and debug the test failure, let's keep it.

Reply:
Ok

Reply:
We also need to keep `Partition Statistics` and `codegenStageIds` for debugging, too?

Reply:
+1 to keep the info.

Reply:
It's the formatted explained plan. So it doesn't really have `Partition Statistics` and `codegenStageIds`.

Reply:
> It's the formatted explained plan. So it doesn't really have Partition Statistics and codegenStageIds.

Yea, just after I wrote the comment, I noticed that soon... Thanks!